app/assets/stylesheets/student_index.scss

```diff
@@ -0,0 +1,54 @@
+@import 'color';
+
+.student_index_container {
+  margin-top: 40px;
+
+  .student_block {
+    margin-right: 10px;
+    margin-bottom: 20px;
+    height: 120px;
+    border-style: solid;
+    border-color: $rock;
+    border-width: 1px;
+
+    background-color: $gray;
+
+    .link_student a {
+      color: $rock;
+      font-weight: bold;
+      font-size: 1.4rem;
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    .link a {
+      color: $brick;
+      font-weight: normal;
+      font-size: 1.3rem;
+      &:hover {
+        color: lighten( $brick, 10% );
+      }
+    }
+
+  }
+
+  .add_student {
+    text-align: center;
+    vertical-align: middle;
+    line-height: 90px;
+  }
+
+  .block_student_form {
+    .name_input {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `name_input` should be written in lowercase with hyphens